nsync/cv_wait_example_test.go

```diff
@@ -87,7 +89,7 @@ func removeAndPrint(q *StringPriorityQueue, delay time.Duration) {
 
 // ExampleMuWait() demonstrates the use of nsync.Mu's Wait() via a priority queue of strings.
 // See the routine RemoveWithDeadline(), above.
-func ExampleCVWait() {
+func ExampleCV_Wait() {
```

Comment:
I'm surprised that go get is recommending the '_'.

Reply:
see: https://blog.golang.org/examples - this is the convention for Type_Method testable examples.